### ISSUE TITLE
getImgIds bug

### ISSUE THREAD

**Issue by ghost** [pdollar/coco#422]
else:
            ids = set(imgIds)
            for i, catId in enumerate(catIds):
                if i == 0 and len(ids) == 0:
                    ids = set(self.catToImgs[catId])
                else:
                    ids &= set(self.catToImgs[catId])

should be changed to 
 
**else:
    ids |= set(self.catToImgs[catId])**

**Comment by HelloDusty** [comment]
yes I find the same bug, I feel you are right and it should be |= not &=
